test/integration/batch_operations_test.lua

```diff
@@ -0,0 +1,1042 @@
+local fio = require('fio')
```

Comment:
In addition to functional tests, we definitely need a performance tests to compare perf of batch vs non-batch operations.

Reply:
Perf tests were added. `crud.insert()` in the loop and `crud.batch_insert()`
are compared for different batch sizes.

Reply:
Now you can see perf test results in the description.